src/types/index.ts

```diff
@@ -255,3 +275,14 @@ export interface IAggregatedWordsElement {
 export type IAggregatedWordsData = IAggregatedWordsElement[];
 
 export type NoData = 'no_data';
+
+export enum MilliSeconds {
```

Comment:
create constant instead

Reply:
Can we leave it as enum?
In case another ms amount will be needed - we'll just expand it